ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/Clusters/ClusterDetailsTable.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { useCallback, useState } from 'react';
+import { generatePath, Link } from 'react-router-dom';
+import {
+    Bullseye,
+    Button,
+    Pagination,
+    SearchInput,
+    Spinner,
+    Text,
+    Title,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import omit from 'lodash/omit';
+
+import { complianceEnhancedScanConfigDetailPath } from 'routePaths';
+import IconText from 'Components/PatternFly/IconText/IconText';
+import useRestQuery from 'hooks/useRestQuery';
+import useURLPagination from 'hooks/useURLPagination';
+import useURLSearch from 'hooks/useURLSearch';
+import useURLSort from 'hooks/useURLSort';
+import {
+    ComplianceCheckStatus,
+    ClusterCheckStatus,
+    getSingleClusterResultsByScanConfig,
+    getSingleClusterResultsByScanConfigCount,
+} from 'services/ComplianceEnhancedService';
+import { SearchFilter } from 'types/search';
+import { SortOption } from 'types/table';
+import { searchValueAsArray } from 'utils/searchUtils';
+
+import TableErrorComponent from 'Containers/Vulnerabilities/WorkloadCves/components/TableErrorComponent';
```

Comment:
Let's add a `TODO` comment to remind us to abstract this out of Vulnerabilities folder